hbp_knowledge/ms/garg2015.bel

```diff
@@ -56,160 +57,176 @@ DEFINE ANNOTATION Species       AS PATTERN "^\d+$"
+#DEFINE ANNOTATION CellStructure AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell-structure/cell-structure-20170511.belanno"
 DEFINE ANNOTATION Cell          AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell/cell-20170511.belanno"
 DEFINE ANNOTATION Confidence    AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/confidence/confidence-1.0.0.belanno"
 DEFINE ANNOTATION Disease       AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/disease/disease-20170511.belanno"
-DEFINE ANNOTATION Gender        AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/gender/gender-1.0.0.belanno"
+#DEFINE ANNOTATION Gender        AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/gender/gender-1.0.0.belanno"
 DEFINE ANNOTATION MeSHAnatomy   AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-anatomy/mesh-anatomy-20170511.belanno"
 DEFINE ANNOTATION MeSHDisease   AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-diseases/mesh-diseases-20170511.belanno"
-DEFINE ANNOTATION Subgraph      AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/neurommsig/neurommsig-1.0.3.belanno"
-DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
+#DEFINE ANNOTATION Subgraph      AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/neurommsig/neurommsig-1.0.3.belanno"
+#DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
 
 ##############
 # Statements #
 ##############
 SET Citation = {"PubMed", "26445701"}
 
 SET Evidence = "Multiple sclerosis is an autoimmune inﬂammatory disorder of CNS of unknown etiology, characterized by demyelination and variable degrees of axonal loss."
-
+SET Confidence = "High"
 path(DOID:"multiple sclerosis") isA path(DOID:"autoimmune disease of central nervous system")
 path(DOID:"multiple sclerosis") -> path(MESH:Inflammation)
 path(DOID:"multiple sclerosis") -> path(HP:"CNS demyelination")
 path(DOID:"multiple sclerosis") -> path(HP:"Axonal loss")
+UNSET Confidence
 
 SET Evidence = " Linkage analysis studies have revealed several gene loci as risk factors, with the major histocompatibility complex (MHC) HLA DR15/DQ6 allele being the strongest one (Barcellos et al. 2003; Sawcer et al. 2011). More recently, alleles of interleukin-2 receptor alpha gene (IL2RA) and interleukin-7 receptor alpha gene (IL7RA) have also been identiﬁed as inheritable risk factors (Haﬂer et al. 2007; Sawcer et al. 2011)."
-
-#path(DOID:"multiple sclerosis") -- bp(MESH:"Major Histocompatilibity Complex")
-#MHC is present in MESH NAMESPACE
-path(DOID:"multiple sclerosis") -- g(HGNC:IL2RA)
-path(DOID:"multiple sclerosis") -- g(HGNC:IL7R)
+#HLA DR15.DQ6 namespace?
```

Comment:
@sgebel kindly help with this statement.

Reply:
use the term g(HGNCGENEFAMILY: "Histocompatibility complex") to code major histocompatibility complex and remove comment from that statement.